From 0b595ee514797c75663a9ddc4ddc81c6c82dc64a Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Sat, 7 Sep 2013 15:11:36 +0300
Subject: [PATCH] do not add default gw as timeserver

*always* adding default gw to timeserver list is a horrible idea

this reverts http://git.kernel.org/cgit/network/connman/connman.git/commit/?id=bbd19813c457227a2861c05e9ebc676f8feb7f10
---
 src/timeserver.c |   13 -------------
 1 files changed, 0 insertions(+), 13 deletions(-)

diff --git a/src/timeserver.c b/src/timeserver.c
index 6440611..6386521 100644
--- a/src/timeserver.c
+++ b/src/timeserver.c
@@ -179,11 +179,9 @@ GSList *__connman_timeserver_add_list(GSList *server_list,
 GSList *__connman_timeserver_get_all(struct connman_service *service)
 {
 	GSList *list = NULL;
-	struct connman_network *network;
 	char **timeservers;
 	char **service_ts;
 	char **service_ts_config;
-	const char *service_gw;
 	char **fallback_ts;
 	int index, i;
 
@@ -204,17 +202,6 @@ GSList *__connman_timeserver_get_all(struct connman_service *service)
 	for (i = 0; service_ts && service_ts[i]; i++)
 		list = __connman_timeserver_add_list(list, service_ts[i]);
 
-	network = __connman_service_get_network(service);
-	if (network) {
-		index = connman_network_get_index(network);
-		service_gw = __connman_ipconfig_get_gateway_from_index(index,
-			CONNMAN_IPCONFIG_TYPE_ALL);
-
-		/* Then add Service Gateway to the list */
-		if (service_gw)
-			list = __connman_timeserver_add_list(list, service_gw);
-	}
-
 	/* Then add Global Timeservers to the list */
 	timeservers = load_timeservers();
 
-- 
1.7.2.5

